### ISSUE TITLE
Support including a remote Ignition config

### ISSUE THREAD

**Issue by crawford** [coreos/bugs#1083]
It would be nice if an Ignition config could include a remote Ignition config. The two would then pass through some sort of function which output a single config. This config would then be applied.

**Comment by crawford** [comment]
It would be especially useful if the config allowed the user to specify the function that processes the configs (e.g. replace, concatenate, merge).

**Comment by crawford** [comment]
This will be fixed once https://github.com/coreos/ignition/pull/183 is merged into master.
